src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: $card-gap;
+  justify-content: center;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media screen and (max-width: $breakpoint-l) and (min-width: $breakpoint-m) {
+    & {
+      grid-template-columns: repeat(3, $card-width);
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-m) and (min-width: $breakpoint-s) {
+    & {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-s) {
+    & {
+      grid-template-columns: $card-width;
+    }
```

Comment:
Мне было бы удобнее читать, если бы брейкпоинты были прописаны абсолютно, а не переменной. Также я бы не использовала ограничение по минимальной и максимальной ширине, а использовала бы что-то одно.

Reply:
Absolutely agree with you.
There is no need for variables here, and it will be easier to read this code, without them.
For the second one, use one condition for media, max or min.

Reply:
Ок, побачив в документації до sass думав повипендрюваться, зараз зміню ))